src/lev-dashboard/DashboardContent.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+import DashboardCounter from "./DashboardCounter";
+
+const CountDiv = styled.div`
+  position: relative;
```

Comment:
Is this still required?

Reply:
Nope, have removed along with the flex direction, which is set as 'row' as default